ecl/hql/hqlexpr.cpp

```diff
@@ -9477,6 +9846,25 @@ CHqlMultiParentScope::CHqlMultiParentScope(IIdAtom * _name, ...)
     va_end(args);
 }
 
+IHqlExpression * CHqlMultiParentScope::lookupNearestSymbol(IIdAtom * searchName, unsigned lookupFlags, HqlLookupContext & ctx, NearestSymbol & nearest)
+{
+    unsigned penalty = 0;
+    NearestSymbol tempNearest(searchName);
+    CHqlScope::lookupNearestSymbol(searchName, lookupFlags, ctx, tempNearest);
+    nearest.compare(tempNearest, penalty);
```

Comment:
ok, so you're right there's probably not much need for it in this instance. I think I had more to this routine that did utilize it but has been cut out. I could unwrap this instance, but the method itself is used in enough place to warrant not removing it and thus it being neater to still use it here. See the comment below for the general idea behind it.